build.md

```diff
@@ -1,52 +1,80 @@
 
-# Building from Source
+# Building locally from source
 
 Prerequisites:
 
 * go version 1.12 or higher is installed and in the path
 * docker is installed and running
 * wget is installed
 
-After setting the `GOPATH` env var correctly, just run `make <action...>` from the command line, within the same directory where `Makefile` resides. For example `make package clean` will run the `package` and then the `clean` actions.
+After setting the `GOPATH` env var correctly, just run `make <action...>` from the command line, within the same directory where `Makefile` resides.
 
+## What is supported by the makefile to build?
 
-Some of the scripts have conditional paths, because certain Linux commands behave differently on OS/X and elsewhere (fun).
```

Comment:
Henry, whilst you're editing this file, please can you change the OS/X references to macOS as we shouldn't be referring to OS/X in our docs? (I do know that you didn't put those references in!)

Reply:
will do